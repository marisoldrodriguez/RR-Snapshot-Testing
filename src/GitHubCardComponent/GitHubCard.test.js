import { render, screen } from '@testing-library/react';
import GitHubCard from './GitGubCard';
import renderer from 'react-test-renderer';

test('renders a snapshot', () => {
    const tree = renderer.create(<GitHubCard />).toJSON()
    expect(tree).toMatchSnapshot()
})
