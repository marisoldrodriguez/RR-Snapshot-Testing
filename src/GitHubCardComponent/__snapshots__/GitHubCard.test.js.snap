// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card"
  >
    <img
      className="card-img-top"
      src="SoftwareDevelopment.png"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
        <h1>
          Marisol D Rodriguez
        </h1>
      </div>
      <p
        className="card-text"
      >
        <h2>
          <a
            href="https://github.com/marisoldrodriguez"
          >
            My GitHub Profile
          </a>
        </h2>
      </p>
    </div>
  </div>
  <br />
  <div
    className="card"
  >
    <div
      className="card-body"
    >
      <p
        className="card-text"
      >
        Experienced product development engineer, fortunate to take a career break to raise my family. Excited to get back in the workforce as a software engineer. Currently enrolled in a Software Development Bootcamp at North Carolina State University. Program completion in October 2022.
      </p>
    </div>
  </div>
</div>
`;
